docs/reference/migration/migrate_query_refactoring.asciidoc

```diff
@@ -71,4 +71,9 @@ codebase and avoid duplication.
 Support for `queryName` and `boost` has been streamlined to all of the queries. That is
 a breaking change till queries get sent over the network as serialized json rather
 than in `Streamable` format. In fact whenever additional fields are added to the json
-representation of the query, older nodes might throw error when they find unknown fields.
+representation of the query, older nodes might throw error when they find unknown fields.pd
+
+==== GeoBoundingBoxQueryBuilder
+
+The type of bounding box query to run can no longer be specified as a string, please
```

Comment:
wondering also cause we did something similar in indices query, probably elsewhere as well.